lib/jellyfish/webhook_notifier.ex

```diff
@@ -0,0 +1,68 @@
+defmodule Jellyfish.WebhookNotifier do
+  @moduledoc """
+  Module responsible for sending notifications to webhooks.
```

Comment:
Sometimes we write `sending notifcations to webhooks` and sometimes  `sending notification through webhooks`. Would be nice to unify this